lib/vehicle_factory.rb

```diff
@@ -0,0 +1,21 @@
+
+
+class VehicleFactory
+    def initialize
+        #
+    end
+
+    def create_vehicles(vehicle_list)
+        output = []
+        vehicle_list.each do |vehicle|
+            output << Vehicle.new({
+                vin: vehicle[:vin_1_10], 
+                year: vehicle[:model_year], 
+                make: vehicle[:make], 
+                model: vehicle[:model], 
+                engine: :ev
+                })
+        end
+        output
```

Comment:
Could also use .map for this and get rid of the accumulator `output`, but this works too!